atlp-rwanda/knights-ecomm-fe
# implement User registration (Vendors and Buyers)

#### What does this PR do?
This pull request implement vendor or buyer user registration

#### Description of Task to be completed?
- Creation of Registration Form Components
- Creation of register related reducers and actions
- Adding register slice in store
- toasting error or success message on register request

#### How should this be manually tested?
- Ensure You have backend service running and set VITE_APP_API_URL in your .env file.
- Clone this repository and navigate to the project directory
    ```
    git clone -b ft-register https://github.com/atlp-rwanda/knights-ecomm-fe.git
    cd knights-ecomm-fe
    npm i
    npm run dev
    ```
access host/register

page UI
![image](https://github.com/atlp-rwanda/knights-ecomm-fe/assets/78812884/58a4a4e7-61e7-461d-a3ce-4eaf3187c282)
